src/main/java/Duke.java

```diff
@@ -1,10 +1,282 @@
+
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    private static final ArrayList<String> Data = new ArrayList<>();
+    private static boolean hasData = false;
+    public static void main(String[] args) throws FileNotFoundException {
+        String LINE  = "_______________________________________________________________\n";
+
+
+
+        Scanner in = new Scanner(System.in);
+
+        //Task[] Task = new Task[100];
+        ArrayList<Task> Task = new ArrayList<>();
+
+        String userInput;
+        String date = "";
+
+        int count = 0;
+
+        String fileName = "C:\\data\\duke.txt"; //handles data file creation
+        try {
+            File myFile = new File(fileName);
+            if (myFile.createNewFile()) {
+                System.out.println("File created: " + myFile.getName() + "\n");
+            } else {
+                System.out.println("File already exists.\n");
+            }
+        } catch (IOException e) {
+            System.out.println("An error occurred.\n");
+            e.printStackTrace();
+        }
+        scanFileContents(fileName); //scans data
+
+        System.out.println(LINE + "Hello! I'm Duke\n" + "Whatchu want\n" + LINE);
+
+        if (hasData) {
+            for (int i = 0; i < Data.size(); i++) { //transfers data into task array
+                if (Data.get(i).contains("[T]")) {
+                    String task = "todo " + Data.get(i).substring(10);
+                    Task.add(count, new Todo(task));
+                    count++;
```

Comment:
Try to avoid arrowhead code and use at most 3 levels of indentation if possible.